rubocop.yml

```
@@ -557,6 +557,9 @@ RSpec/ContextWording:
 Naming/HeredocDelimiterNaming:
   Enabled: false
 
+Naming/MethodParameterName:
+  MinNameLength: 2
```

I probably favour adding `is` to the list of [`AllowedNames`](https://docs.rubocop.org/rubocop/cops_naming.html#configurable-attributes-11).
ie. (untested)
```suggestion
  AllowedNames: ['at', 'by', 'db', 'id', 'in', 'io', 'ip', 'of', 'on', 'os', 'pp', 'to', 'is']
```

---

Thanks!
Tested and it works.

[Before](https://github.com/voxpupuli/puppet-mongodb/actions/runs/1968591285)
[After](https://github.com/voxpupuli/puppet-mongodb/actions/runs/1973363472)